app/jobs/course/assessment/submission/unsubmitting_job.rb

```diff
@@ -5,29 +5,31 @@ class Course::Assessment::Submission::UnsubmittingJob < ApplicationJob
 
   protected
 
-  def perform_tracked(unsubmitter, submission_ids, assessment)
+  def perform_tracked(unsubmitter, submission_ids, assessment, question = nil, redirect_to_path = nil)
```

Comment:
Can you add a comment/method documentation above this explaining the params and that this method also optionally handles the deletion of a question's answers?

Reply:
Also, can you add class documentation for this `UnsubmittingJob` as well, i.e. above line 2, since it's not explicit that this job also does deletion.